src/blocks/net.rs

```diff
@@ -195,6 +195,14 @@ pub struct NetConfig {
     /// Whether to hide networks that are down/inactive completely.
     #[serde(default = "NetConfig::default_hide_inactive")]
     pub hide_inactive: bool,
+
+    /// Whether to show the upload throughput of active networks
+    #[serde(default = "NetConfig::default_tx")]
+    pub tx: bool,
```

Comment:
I don't think it's really necessary to have two configuration options -- I can't imagine anyone wants to see *just* up or *just* down speeds. Why not simplify this to a single `speed` option (which mirrors the language of i3blocks)?

Reply:
I can imagine. For example, I have a reasonably stable uplink, but my downlink can vary immensely